dev/protos/protos.json

```diff
@@ -0,0 +1,5640 @@
+{
```

Comment:
Are we still using the file? I wanted to get rid of it so that the clients load only the part of the proto data that they need.

Reply:
This should go away with https://github.com/googleapis/nodejs-firestore/pull/1677

Reply:
Add `unlink` in owlbot.py in this PR